packages/lib/i18n.ts

```diff
@@ -12,3 +12,28 @@ export const localeOptions = locales.map((locale) => ({
 export const defaultLocaleOption = localeOptions.find(
   (locale) => locale.value === i18n.defaultLocale
 ) as (typeof localeOptions)[number];
+
+interface ExtendedLocale extends Intl.Locale {
+  getTextInfo?: () => { direction: "ltr" | "rtl" };
+}
+
+export function getTextDirection(locale: string): "ltr" | "rtl" {
+  try {
+    if (typeof Intl.Locale !== "undefined" && "getTextInfo" in Intl.Locale.prototype) {
+      const extendedLocale = new Intl.Locale(locale) as ExtendedLocale;
+      if (extendedLocale.getTextInfo) {
+        return extendedLocale.getTextInfo().direction;
+      }
+    }
+    // Fallback for gbrowsers that don't support Intl.Locale.prototype.getTextInfo
+    const rtlLocales = ["ar", "he"];
+    return rtlLocales.some((lang) => locale.toLowerCase().startsWith(lang)) ? "rtl" : "ltr";
```

Comment:
```suggestion
    const locale = new Intl.Locale(locale);
    if (locale.textInfo.direction) rerturn locale.textInfo.direction;
    // fallback
    const rtlLocales = ["ar", "he"];
    return rtlLocales.some((lang) => locale.toLowerCase().startsWith(lang)) ? "rtl" : "ltr";
```

Do something like this and rewrite this

Reply:
hi again, I made a [commit](https://github.com/calcom/cal.com/pull/16697/commits/feb168b5d9edbe6b0ed66034170cb13f3bc536c8) that solves this